Resources/public/js/views/ez-dashboardblocksview.js

```diff
@@ -145,7 +145,14 @@ YUI.add('ez-dashboardblocksview', function (Y) {
              * @readOnly
              */
             blocks: {
-                value: [],
+                valueFn: function () {
+                    return [
+                        new Y.eZ.DashboardBlockAllContentView({
+                            priority: 1,
+                            identifier: 'everyones-content'
+                        }).addTarget(this).set('active', this.get('active'))
```

Comment:
Good idea. I didn't know about the `bubbleTargets` as it wasn't very well documented in YUI docs.